src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -474,7 +475,7 @@ _test_topology_invalidate_server (bool pooled)
    checks_init (&checks);
    uri = test_framework_get_uri ();
    /* no auto heartbeat */
-   mongoc_uri_set_option_as_int32 (uri, "heartbeatFrequencyMS", INT32_MAX);
+   mongoc_uri_set_option_as_int32 (uri, "heartbeatFrequencyMS", 60000);
```

Comment:
I need to verify this, but setting INT32_MAX results in an immediate network error when sending the awaitable ismaster.